scripts/zkai_attention_kv_risc0_sequence_receipt_gate.py

```
@@ -0,0 +1,891 @@
+        input_path.write_text(json.dumps(expected_input(), sort_keys=True) + "\n", encoding="utf-8")
+        return run_host("verify", input_path, receipt_path, summary_path)
+
+
+def build_payload(*, prove: bool = False, receipt_path: pathlib.Path = RECEIPT_OUT, previous_proof_generation_time_ms: float | None = None) -> dict[str, Any]:
+    journal = expected_journal()
+    receipt_path = _resolved_under_root(receipt_path, label="receipt", layer="output_path")
+    toolchain = require_available_toolchain()
+    host_summary = generate_or_verify_receipt(prove=prove, receipt_path=receipt_path)
+    receipt_bytes = receipt_path.read_bytes()
+    if not receipt_bytes or len(receipt_bytes) > MAX_RECEIPT_BYTES:
+        raise AttentionKvRisc0SequenceReceiptError("receipt artifact size outside allowed bound", layer="receipt_artifact")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)